modules/ui/src/main/scala/lucuma/ui/package.scala

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui
+
+import japgolly.scalajs.react.vdom.VdomNode
+import react.common.*
+import react.primereact.Message
+import react.primereact.ProgressSpinner
+
+val DefaultPendingRender: VdomNode = ProgressSpinner(clazz = Css("pending-loader"))
```

Comment:
Since this is likely called from many places would it be worth to make the `Css` instance a separate constant?

Reply:
For the moment, this was the only reference to it. We can move it to a constant if it's needed from somewhere else.